packages/sdk-vue/src/types.ts

```diff
@@ -1,5 +1,5 @@
 import { Ref } from 'vue';
-import { UseCookie } from './createFusionAuth/NuxtUseCookieAdapter';
+import { useCookie as useCookieType } from 'nuxt/app';
```

Comment:
We don't need to `import as` here because there's no name collision in this file.